pkg/apitestsv2/apitest_test.go

```diff
@@ -67,17 +67,9 @@ func TestAPITest_Invoke(t *testing.T) {
 				testEnv:    &apistructs.APITestEnvData{},
 				caseParams: nil,
 			},
-			want: &apistructs.APIRequestInfo{
-				URL:     "http://www.erda.cloud",
-				Method:  "GET",
-				Headers: map[string][]string{"Accept-Encoding": {"identity"}},
-				Params:  map[string][]string{},
-				Body: apistructs.APIBody{
-					Content: "",
-				},
-			},
+			want:    nil,
 			want1:   &apistructs.APIResp{},
-			wantErr: false,
+			wantErr: true,
```

Comment:
why err

Reply:
This one side will send a request, which cannot test whether the startup agent can send successfully, so only the err case is tested